modules/system-info/reporters/server.php

```diff
@@ -300,6 +300,8 @@ public function get_write_permissions() {
 		$paths_to_check = [
 			ABSPATH => 'WordPress root directory',
 		];
+		
+		$paths_to_check = apply_filters('elementor/modules/system-info/reporters/server/initial_paths_to_check', $paths_to_check);
```

Comment:
It makes more sense to do this in line 325

Reply:
I assume you actually meant [line 323](https://github.com/elementor/elementor/blob/master/modules/system-info/reporters/server.php#L323)? 

Sure? I can move the filter to occur right before you start looping through the array, if that's how you would prefer it. I just need some way to modify the list.

Reply:
Yes I believe editing it once it's fully populated is the way to go.

Reply:
Done. Please check updated version

Reply:
@gilzow Please fix PHP Lint issues, see CI test runs at the bottom of the Conversation screen in this PR